src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -123,17 +124,8 @@ const ViewPopup = (() => {
             RUNLOW      : localize('Only Downs'),
             MULTUP      : localize('Multiplier Up'),
             MULTDOWN    : localize('Multiplier Down'),
-        });
-
-        return {
-            get: () => {
-                if (!contract_type_display) {
-                    contract_type_display = initContractTypeDisplay();
-                }
-                return contract_type_display;
-            },
```

Comment:
You need to revert this back, this is how we define basic functionaries for each of the components.

Reply:
ContractTypeDisplay.get() is used in multiple place as well.

Reply:
Upon checking... `ContractTypeDisplay()` only gets used in one place.

I made the change because it was written as `IIFE` initially and `ContractTypeDisplay.get()` gets executed once. Whenever a user clicks on the menu to change to between `Higher/Lower` and `Rise/Fall` the `IIFE` doesn't give a chance to execute logic that involves change in `formname`.